js/src/lib/components/InferenceWidget/widgets/ReinforcementLearningWidget/ReinforcementLearningWidget.svelte

```diff
@@ -0,0 +1,67 @@
+<script lang="ts">
+	import type { WidgetProps } from "../../shared/types";
+	import { onMount } from "svelte";
+	import IconSpin from "../../../Icons/IconSpin.svelte";
+	import WidgetHeader from "../../shared/WidgetHeader/WidgetHeader.svelte";
+
+	export let apiToken: WidgetProps["apiToken"];
+	export let model: WidgetProps["model"];
+
+	enum Replay {
+		Loading,
+		Available,
+		Unavailable,
+	}
+
+	let replay = Replay.Loading;
+
+	function doesReplayExist() {
+		// check if repository has `replay.mp4` file
+		const replayUrl = `https://huggingface.co/${model.id}/resolve/main/output.mp4`;
+		const xhr = new XMLHttpRequest();
+		xhr.open("HEAD", replayUrl, false);
+		xhr.send();
+		return xhr.status !== 404;
+	}
+
+	onMount(() => {
+		replay = doesReplayExist() ? Replay.Available : Replay.Unavailable;
+	});
+</script>
+
+<!-- 
+	WidgetWrapper.svelte component is not used because this pipeline widget (reinforcement-learning)
+	does NOT use InferenceAPI (unlike other pipelines widgets)
+-->
+<div class="flex flex-col w-full max-w-full">
+	<WidgetHeader noTitle pipeline="reinforcement-learning" />
+	<div class="rounded-lg overflow-hidden w-full">
+		{#if replay === Replay.Available}
+			<!-- svelte-ignore a11y-media-has-caption -->
+			<video
+				class="w-full"
+				src="https://huggingface.co/ThomasSimonini/ppo-LunarLander-v2/resolve/main/output.mp4"
```

Comment:
should not be hardcoded no?

Reply:
fixed the hardcode https://github.com/huggingface/hub-docs/pull/75/commits/baee87f938b2e13eeaf212ccc6930b62f7ad8496